browser/components/pages/MetricsPage/index.js

```diff
@@ -43,16 +44,16 @@ export default class MetricsPage extends Component {
   render(){
     const week = this.parseWeek()
     const metrics = this.props.metrics && this.props.metrics[formatWeek(week)]
+    const  { prrrsPerWeek ={} } = this.props
     return <Layout className="MetricsPage" session={this.props.session}>
       <WeekBar week={week} />
       <h2 className="MetricsPage-header">Metrics For: {formatWeek(week)}</h2>
       <Metrics metrics={metrics} />
+      <Timeline week={week} prrrs={metrics ? metrics.prrrs : null} />
```

Comment:
I know I wrote this but I think I like this more
`<Timeline week={week} prrrs={metrics && metrics.prrrs} />`